src/pages/FolderPage.js

```diff
@@ -0,0 +1,42 @@
+import './FolderPage.css';
+import Search from '../components/Search';
+import Card from '../components/Card';
+import FolderOwner from '../components/FolderOwner';
+import { useEffect, useState } from 'react';
+import { getMockFolder } from '../api';
+import useAsync from '../hooks/useAsync';
+
+function FolderPage() {
+  const [, , getUserFolder] = useAsync(getMockFolder);
+  const [folder, setFolder] = useState([]);
+
+  const getFolder = async () => {
+    const result = await getUserFolder();
+    if (!result) return;
+
+    const {
+      folder: { links },
+    } = result;
+    setFolder(links);
+  };
+
+  useEffect(() => {
+    getFolder();
+  }, []);
+
+  return (
+    <main className="main_wrap">
+      <FolderOwner />
+      <Search />
+      <ul className="folder">
+        {folder.map((item) => {
+          const { createdAt, url, title, description, imageSource } = item;
+          const initialValue = { createdAt, url, title, description, imageSource };
+          return <Card key={item.id} initialValue={initialValue} preview={imageSource} />;
```

Comment:
33 라인에서 item 안에 있는 프로퍼티들을 풀어서 
34 라인에서 다시 객체로 묶은 후 35라인에서 props 로 전달하는 이유가 있을까요??
아래와 같이 바로 넘겨도 동일해보여요! 

```suggestion
          return <Card key={item.id} initialValue={item} preview={imageSource} />;
```